README.md

```diff
@@ -1,2 +1,13 @@
 # flytekit-python-template
 A template for the recommended layout of a Flyte enabled repository for code written in python using flytekit
+
+## Usage
+To get up and running with your Flyte project, we recommend proceeding through the following:
+
+1. Deploy the flyte sandbox locally by following the instructions at [getting started guide](https://flyte.readthedocs.io/en/latest/administrator/install/getting_started.html#getting-started)
```

Comment:
Flyte Sandbox locally or to any Kubernetes Cluster (EKS, GKE, custom etc)

Reply:
done